editor/project_settings_editor.cpp

```diff
@@ -145,12 +154,12 @@ void ProjectSettingsEditor::_action_edited() {
 	if (new_name == old_name)
 		return;
 
-	if (new_name.find("/") != -1 || new_name.find(":") != -1 || new_name.find("\"") != -1 || new_name == "") {
+	if (new_name == "" || !_validate_action_name(new_name)) {
 
 		ti->set_text(0, old_name);
 		add_at = "input/" + old_name;
 
-		message->set_text(TTR("Invalid action (anything goes but '/', ':' or '\"')."));
+		message->set_text(TTR("Can't contain '/', ':', '=', '\\' or '\"'"));
```

Comment:
I know understand why the messages were different, since this one also needs to handle the case `new_name == ""`, which was covered by "Invalid action".

We can still use the same for both, but maybe use `Invalid action name. It cannot be empty nor contain '/', ':', '=', '\\' or '\"'`.